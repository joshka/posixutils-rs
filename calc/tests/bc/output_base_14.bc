obase = 14
0
1
123.790
0.7893
quit
