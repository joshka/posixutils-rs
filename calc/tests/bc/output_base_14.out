0
1
8B.B0B
0.B09B
